hedera-mirror-rest/controllers/contractController.js

```diff
@@ -748,8 +784,24 @@ class ContractController extends BaseController {
     if (rows.length !== 1) {
       throw new NotFoundError();
     }
-
-    res.locals[responseDataLabel] = formatContractRow(rows[0], ContractBytecodeViewModel);
+    const contractRows = rows;
```

Comment:
```suggestion
    const contract = rows[0];
```

and use it hereafter.